benchmark/httpapi.py

```diff
@@ -251,20 +368,59 @@ def create_api_service(endpoint):
     :return: Service that will listen on the endpoint using HTTP API server.
     """
     api_root = Resource()
-    api = BenchmarkAPI_V1(InMemoryBackend())
+    api = BenchmarkAPI_V1(backend)
     api_root.putChild('v1', api.app.resource())
 
     return StreamServerEndpointService(endpoint, Site(api_root))
 
 
+class BackendService(Service):
+    """
+    A basic Twisted service that wraps the peristence backend.
+    """
+    def __init__(self, backend):
+        super(Service, self).__init__()
+        self.backend = backend
+
+    def stopService(self):
+        return self.backend.disconnect()
+
+
+def start_services(reactor, endpoint, backend):
+    top_service = MultiService()
+    api_service = create_api_service(endpoint, backend)
+    api_service.setServiceParent(top_service)
+    backend_service = BackendService(backend)
+    backend_service.setServiceParent(top_service)
+
+    # XXX Make startService() raise an exception on an error
+    # instead of just logging and dropping it.
```

Comment:
I don't know if this comments means that the code does not do this but we should change it to do so in the future or whether it describes what it does. If the latter, please remove the 'XXX'.